prime

```diff
@@ -0,0 +1,22 @@
+#include<iosteam>
+using namespace std;
+int main{
+    int n;
+    cout<<"enter a number"<<endl;
```

Comment:
No need to add cout statement as this code will be used as a helper code. So no print statements apart form input/output

Reply:
Pleaser resolve above comments